rest/common/utils/rest.py

```diff
@@ -6,26 +6,32 @@
 import time
 
 import requests
-from dotenv import dotenv_values
 
 from rest.common.config.config import Config
 from rest.common.utils.exceptions import CommandExecutionError, HTTPError
 from utility.log import Log
 
+# from dotenv import dotenv_values
+
+
 log = Log(__name__)
 
 
-def rest():
+def rest(**kw):
```

Comment:
is this parameter optional?
We are using this rest.py in other NVME scripts also where no arguments are passed while calling the function.
Will it not regress them?

Some other files where this has been used:
1) [tests/nvmeof/test_nvmeof_rest_sanity.py](https://github.com/red-hat-storage/cephci/blob/2dd8e6480224663af1697ba62b61e702bb49191c/tests/nvmeof/test_nvmeof_rest_sanity.py#L27)
2) [tests/nvmeof/rest/test_nvmeof_rest_sanity.py](https://github.com/red-hat-storage/cephci/blob/2dd8e6480224663af1697ba62b61e702bb49191c/tests/nvmeof/rest/test_nvmeof_rest_sanity.py#L25)